src/modules/getScores.js

```diff
@@ -0,0 +1,5 @@
+const getScore = () => fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/HX42ejstWWGWDKZzkjqD/scores/')
+  .then((response) => response.json())
+  .then((data) => data);
+
+export default getScore;
```

Comment:
- [ ] Instead of using `.then` please consider using `async` and `await` to handle asynchronous requests! 🤓 🔧 

According to the requirements:
![image](https://user-images.githubusercontent.com/77069011/219508775-20d8d353-b7d7-4b2e-b1a8-f762f8b90dce.png)

Reply:
I have used async and await now.